Wikipedia/Code/PreviewAndSaveViewController.m

```diff
@@ -382,7 +381,10 @@ - (void)showSummaryOverlay {
     // Set the overlay's text field to self.summaryText so it can display
     // any existing value (in case user taps "Other" again)
     summaryVC.summaryText = self.summaryText;
-    summaryVC.previewVC = self;
```

Comment:
I refactored the way `summaryText` is injected into `PreviewAndSaveViewController`.

I did so to prevent `PreviewAndSaveViewController` being injected into `EditSummaryViewController` which creates a tight coupling between these classes. It helps in reading the code (reduces cognitive load) and helps in making `EditSummaryViewController` reusable.